remindmoi_bot_handler.py

```diff
@@ -133,6 +143,31 @@ def get_bot_response(message: Dict[str, Any], bot_handler: Any) -> str:
                 [f"@**{email}**" for email in response["user_emails_to_remind"]]
             )
             return f"Reminder will be sent to {emails}. Your reminder id is: {response['reminder_id']}."
+        if is_calendar_remind_command(message_content):
```

Comment:
Remember that the calendar needs to be stored by all commands, specially for `remindme` one

Reply:
but I have a History model to this reminder and it's saved on calendar, it's necesary to save it also in another model? the id's in this cases can't be used to modify the event